content/using-atmosphere-packages.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Using Atmosphere Packages
+order: 28
+discourseTopicId: 20192
+---
+
+Atmosphere packages are packages written specifically for Meteor and have several advantages over npm when used with Meteor. In particular, Atmosphere packages can:
```

Comment:
I want to keep things in line with the rest of the guide - is it bad practice to have a <h2> as the first thing in a section? i.e. Should there always be some introductory text?